app/controllers/v0/ppiu_controller.rb

```diff
@@ -3,6 +3,7 @@
 module V0
   class PPIUController < ApplicationController
     before_action { authorize :evss, :access? }
+    before_action { authorize :ppiu, :access? }
```

Comment:
Can we get a spec showing that a 403 is returned if the user does not have ppiu access?
Currently, if you remove this line, all the specs still pass.